options/options.cpp

```diff
@@ -587,6 +591,37 @@ const option::Descriptor usage[] = {
     Arg::Numeric,
     "  --kind-bound-step \tAmount by which bound (unrolling depth) is "
     "increased in k-induction (default: 1)" },
+  { NO_INTERP_FRONTIER_SIMPL,
+    0,
+    "",
+    "no-interp-frontier-simpl",
+    Arg::None,
+    "  --no-interp-frontier-simpl \tDisable frontier-set simplification in "
+    "interp engine" },
+  { INTERP_PROPS,
+    0,
+    "",
+    "interp-props",
+    Arg::NonEmpty,
+    "  --interp-props \tSpecifies at which time frames properties are "
+    "considered when computing interpolants: all, first-and-last, only-last "
+    "(WARNING: choosing not 'all' could cause incompleteness on some "
```

Comment:
incompleteness in the case of QF_BV / QF_ABV benchmarks, right?
What about other theories?

Reply:
Also, what result is reported when using only-last?

Reply:
I think we should produce a fix point that contains bad states -- this causes confusion because we cannot say that to be an invariant strong enough to be a proof witness.

I think we should remove that case.

Reply:
Done in c82afe0